crates/services/consensus_module/poa/src/service.rs

```diff
@@ -281,9 +345,8 @@ where
                         .await;
                 }
             }
-            (Trigger::Interval { .. }, RequestType::Manual)
-            | (Trigger::Hybrid { .. }, RequestType::Manual) => {
-                unreachable!("Trigger types interval and hybrid cannot be used with manual. This is enforced during config validation")
+            (Trigger::Hybrid { .. }, RequestType::Manual) => {
+                unreachable!("Trigger types hybrid cannot be used with manual. This is enforced during config validation")
```

Comment:
Is this still the case though? Seems like `next_time(..)` would just use the `min_block_time` as the interval.

Reply:
Yeah, we can remove it, but I didn't do that to minimize the number of cases that we need to support=D